gyp/pylib/gyp/input.py

```diff
@@ -911,9 +911,7 @@ def ExpandVariables(input, phase, variables, build_file):
           p_stdout, p_stderr = p.communicate('')
```

Comment:
For consistency, my preference would be the solution we use elsewhere:
```
if bytes != str:  # Python 3
    p_stdout = p_stdout.decode('utf-8')
    p_stderr = p_stderr.decode('utf-8')
```
And then revert the change to line 914.

Reply:
#1937